components/ShippingRates/RateTable.vue

```diff
@@ -78,7 +78,7 @@ const props = defineProps({
     type: Boolean,
     default: false
   },
-  readOnly: {
+  isClassView: {
```

Comment:
i think `isClassView` is too specify
how much `isEditable`?

Reply:
but they all editable 🤔